config.private.json

```diff
@@ -1,3 +1,4 @@
 {
-  "api": "https://wwwint.ebi.ac.uk/metagenomics/api/v1/"
+  "api": "https://wwwint.ebi.ac.uk/metagenomics/api/v1/",
+  "googleMapsKey": "AIzaSyBUT4xxlaKgoU5m98wjvHkFZGIcWjpgJAU"
```

Comment:
Whilst this key is embedded in the website builds and therefore not "secret", it is still best practice to not check it into GitHub.

Reply:
@SandyRogers I've pushed a commit to remove this